terraform/cloudfoundry/dns-records.tf

```diff
@@ -82,6 +82,14 @@ resource "aws_route53_record" "logsearch" {
   records = ["${aws_elb.logsearch_kibana.dns_name}"]
 }
 
+resource "aws_route53_record" "logsearch_ingestor" {
+  zone_id = "${var.system_dns_zone_id}"
+  name    = "logsearch-ingestor.${var.system_dns_zone_name}."
+  type    = "CNAME"
```

Comment:
Ok, but in this case I think it's better to be consistent. If we want to use alias records, then that's a change that we should make to all records pointing at ELBs. Having a mixture will cause people reading this later to have to try to figure out why some use an alias and some use a cname.